packages/gatsby-cli/src/reporter/index.ts

```diff
@@ -3,7 +3,11 @@ import { patchConsole } from "./patch-console"
 import { catchExitSignals } from "./catch-exit-signals"
 import { reporter } from "./reporter"
 
-catchExitSignals()
+// Boy, I'm going to regret this
+if (process.argv[2] !== `develop`) {
```

Comment:
I added this as a hack because:

When two threads are spun up (this and the child process), they both import reporter which initialises packages/gatsby-cli/src/reporter/catch-exit-signals.ts twice 

This means that there is now a duplicate exit handler 🤦 which results in multiple `SET_STATUS` messages sent back over IPC

This hack makes it so that the exit handler is only set up for the parent process. 

Cc @pieh 😂

Reply:
This doesn't affect consumers of structured logs (Cloud)